airflow/operators/sql.py

```diff
@@ -34,18 +34,26 @@ class BaseSQLOperator(BaseOperator):
     You can custom the behavior by overriding the .get_db_hook() method.
     """
 
-    def __init__(self, *, conn_id: Optional[str] = None, database: Optional[str] = None, **kwargs):
+    def __init__(
+        self,
+        *,
+        conn_id: Optional[str] = None,
+        database: Optional[str] = None,
+        hook_params: Dict = {},
```

Comment:
This is wrong, see the important warning on this page: https://docs.python.org/3/tutorial/controlflow.html#default-argument-values